Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -965,7 +873,8 @@ static u32 TranslateBlockAddress(const u32 addr, const XCheckTLBFlag _Flag)
 }
 
 // Translate effective address using BAT or PAT.  Returns 0 if the address cannot be translated.
-u32 TranslateAddress(const u32 _Address, const XCheckTLBFlag _Flag)
+template <const XCheckTLBFlag _Flag>
```

Comment:
You might be able to get away without it, but it would be good documentation to explicitly instantiate all 3 (?) versions of this function.